test/inductor/test_mkldnn_pattern_matcher.py

```diff
@@ -778,7 +798,16 @@ def test_qconv2d_relu_cpu(self):
         r"""
         This testcase will quantize Conv2d->ReLU pattern.
         """
-        self._qconv2d_unary_cpu_test_helper()
+        self._qconv2d_unary_cpu_test_helper(device="cpu")
+
+    @skipIfNoDynamoSupport
+    @skipIfNoONEDNN
+    @skipIfNoXPU
+    def test_qconv2d_relu_xpu(self):
+        r"""
+        This testcase will quantize Conv2d->ReLU pattern.
+        """
+        self._qconv2d_unary_cpu_test_helper(device="xpu")
```

Comment:
The function name is confusing. `_qconv2d_unary_cpu_test_helper` -> `_qconv2d_unary_test_helper`

Reply:
thanks for reminding, these names have been changed.